app/grants/views.py

```diff
@@ -110,6 +125,12 @@ def grants(request):
     if grant_stats.exists():
         grant_amount = grant_stats.first().val / 1000
```

Comment:
maybe do the formatting to the `$x k` amount in the frontend instead of here? Just thinking if someone else tries to use this context var they might be confused... probably doesn't matter either way though.... what do you think?